src/main/java/org/jitsi/srtp/crypto/Aes.java

```diff
@@ -433,22 +404,17 @@ else if (t instanceof ThreadDeath)
                 {
                     if (factory == null)
                     {
-                        factoryAndTimestamp = Aes.fastestFactories
+                        factory = Aes.fastestFactories
                             .getOrDefault(transformation, null);
```

Comment:
Ideally you'd use DEFAULT_FACTORY instead of null here directly and get rid of the next check.